hammer.1

```diff
@@ -0,0 +1,160 @@
+.\" Manpage for hammer cli.
+.\" Contact domcleal@redhat.com to correct errors or typos.
+.TH man 1 "11 March 2015" "1.4" "hammer man page"
```

Comment:
I don't think this file should be stored in this format, as the date, version number etc can be autogenerated, as can the difficult formatting.